training_record/serializers.py

```diff
@@ -55,7 +55,7 @@ class Meta:
 
     def create(self, validated_data):
         return CampusEventFeedbackService.create_feedback(
-            **validated_data)
+            self.context, **validated_data)
```

Comment:
`create_feedback(**validated_data, context=self.context)`
Usually, we put variables for business logic before context variables.